docs/developer.rst

```diff
@@ -104,8 +104,7 @@ Then be sure to pass the tests::
 
 When contributing to pycodestyle, please observe our `Code of Conduct`_.
 
-To run the Travis CI tests, you'll have to create a build the pycodestyle
-package using tox::
+To run the tests, use tox::
```

Comment:
To me, the intend is "run tests". Instead of running test "the way Travis CI does", CI should be running the tests as *we* do.